README.md

```diff
@@ -26,14 +26,14 @@ lightline plugin:
   programs.nixvim = {
     enable = true;
 
-    colorschemes.gruvbox.enable = true;
-    plugins.lightline.enable = true;
+    colorschemes.catppuccin.enable = true;
+    plugins.lualine.enable = true;
   };
 }
 ```
 
-When we do this, lightline will be set up to a sensible default, and will use
-gruvbox as the colorscheme, no extra configuration required!
+When we do this, lualine will be set up to a sensible default, and will use
+catppuccin as the colorscheme, no extra configuration required!
```

Comment:
Is this still true? If so we should change this behaviour given lightline obviously doesn't work the way we thought it did when we initially implemented that behaviour...

If it isn't true anymore, we are fine to just amend the documentation.

Reply:
Ah sorry, I'm getting mixed up between lightline and lualine. Can you still confirm that we don't auto-set colorschemes for lightline?

Reply:
The behavior of lightline is to use their default color scheme. Regardless of what my neovim color scheme is, lightline displays the same color scheme.

Reply:
I think we talked about it elsewhere, but it appears in their documentation that its very much intentional to not automatically utilize the edit colorscheme: https://github.com/itchyny/lightline.vim/blob/58c97bc21c6f657d3babdd4eefce7593e30e75ce/README.md?plain=1#L130

Reply:
> The behavior of lightline is to use their default color scheme. Regardless of what my neovim color scheme is, lightline displays the same color scheme.

I understand that's the upstream plugin behaviour, however we used to set `plugins.lightline.colorscheme` to `config.colorscheme` by default, which lead to #1934 being reported: https://github.com/nix-community/nixvim/blob/e48ce785d9e72c0106319d93e23c5579336ffe33/plugins/statuslines/lightline.nix#L19-L24

It seems this is no longer the case, so my concern is resolved: https://github.com/nix-community/nixvim/blob/7e3ed24e52e2d400b84b22524cba987a495a3f24/plugins/statuslines/lightline.nix#L54-L60

> it appears in their documentation that its very much intentional to not automatically utilize the editor colorscheme

Yes I remember that discussion. Interestingly, digging through older discussions it seems that lightline _does_ support setting `plugins.lightline.settings.colorscheme = "gruvbox"` _if_ a different gruvbox plugin is also installed, see this thread: https://github.com/nix-community/nixvim/discussions/1575#discussioncomment-10254132

Reply:
> > The behavior of lightline is to use their default color scheme. Regardless of what my neovim color scheme is, lightline displays the same color scheme.
> 
> I understand that's the upstream plugin behaviour, however we used to set `plugins.lightline.colorscheme` to `config.colorscheme` by default, which lead to #1934 being reported:
> 

Sorry, that makes more sense... I forgot that we did that. That was a long 2 days ago :P 

> > it appears in their documentation that its very much intentional to not automatically utilize the editor colorscheme
> 
> Yes I remember that discussion. Interestingly, digging through older discussions it seems that lightline _does_ support setting `plugins.lightline.settings.colorscheme = "gruvbox"` _if_ a different gruvbox plugin is also installed, see this thread: [#1575 (reply in thread)](https://github.com/nix-community/nixvim/discussions/1575#discussioncomment-10254132)

That is interesting to me.. I did see some talk about that but, I wasn't sure how it was supported based on what I was reading on their github.